proto/skaffold.proto

```diff
@@ -27,11 +27,13 @@ message State {
     FileSyncState fileSyncState = 6;
     repeated DebuggingContainerEvent debuggingContainers = 7;
     Metadata metadata = 8;
+    TestState testState = 9;
 }
 
 message Metadata {
     BuildMetadata build = 1;
     DeployMetadata deploy = 2;
+    TestMetadata test = 3;
```

Comment:
How are we setting the `TestMetadata`?  is there code to add it?

Reply:
These are just proto changes for now.